spec/ibm/cloud/sdk/vpc_spec.rb

```diff
@@ -23,10 +23,13 @@
   vpn_gateways: IBM::Cloud::SDK::VPC::VPNGateways
 }.freeze
 
+
+vpc = IBM::CloudSDK.new(ENV['IBM_CLOUD_APIKEY']).vpc
```

Comment:
We should probably mock the API calls so the tests can run on travis without an actual API Key

Reply:
I opened an issue to track the work for the failing specs, is going to be too much for this pr so we can do it in a follow-up